src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,149 @@
+      $ref: './fields.yaml#/created_at'
+    updated_at:
+      $ref: './fields.yaml#/updated_at'
+    file_name:
+      type: string
+      description: |
+        Specifies the name of file uploaded by the user.
+      example: amex.csv
+    file_size:
+      type: number
+      description: |
+        Specifies the size of file uploaded by the user. (The size is represented in bytes)
+      example: 21
+    no_of_debit:
+      type: integer
+      description: |
+        Number of debit transactions in the statement uploaded by the user.
+      example: 3
+    no_of_credit:
+      type: integer
+      description: |
+        Number of credit transactions in the statement uploaded by the user.
+      example: 3
+    no_of_transactions:
+      type: integer
+      description: |
+        Number of transactions in the statement uploaded by the user.
+      example: 6
+    statement_columns:
+      type: array
+      items: 
+        type: string
+      description: >
+        This specifies headers of the statement uploaded by the user.
+      example:  
+        - account_number
+        - amount
+    start_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Start date of the statement period
+      nullable: true
+    end_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        End date of the statement period
+      nullable: true
+    has_statement_duplicate:
+      type: boolean
+      description: |
+        Indicates whether the statement is completely duplicate or not.
+    duplicate_statement_ids:
+      type: array
+      description:
+        Holds the list of statement_ids which are duplicate to the current statement
+      items:
+        $ref: './fields.yaml#/id_string'
+      example:
+        - stmtge49ielgel
+      nullable: true
+    has_statement_line_items_duplicate:
+      type: boolean
+      description: |
+        Indicates whether there are any line items in this statement are duplicates.
+    duplicate_statement_line_items:
+      type: array
+      items:
+        type: object
+        properties:
+          ids:
+            type: array
+            items:
+              $ref: './fields.yaml#/id_string'
+            example:
+              - stlige49ielgel
+            description:
+              Ids of the current statement line items which are duplicates
+            nullable: true
+          duplicate_line_item_statement_id:
+            type: string
+            description: |
+              Id of the existing duplicate statement
+            example:
+              - stmpge49ielgel
+            nullable: true
+    unique_id:
+      type: string
+      description: |
+        Unique key that is used to identify the statement.
+      example: 74175aadf22675aa59df28b96d351cfb
+    file_id:
+      type: string
+      description: |
+        File id of the uploaded statement.
+      example: fiit3pX9gzzP
+    is_corporate_card_transactions_created:
```

Comment:
Have updated this, but still not confirmed here -> if some other suggestion comes in there, will update here as well -> https://fylein.slack.com/archives/C02PYUNBQRE/p1652691640725639?thread_ts=1652679368.966629&cid=C02PYUNBQRE